pelican/server.py

```diff
@@ -99,6 +100,9 @@ def guess_type(self, path):
 
         return mimetype
 
+    def log_message(self, format, *args):
+        logger.info("%s" % format % args)
```

Comment:
Extra formatting is unnecessary. This should be enough:
```python
logger.info(format, *args)
```

Reply:
Fixed! The previous code is what is used upstream.